docs/VARIABLES.md

```diff
@@ -2380,6 +2380,86 @@ Default:  "{{mds_ssl_enabled}}"
 
 ***
 
+### sso_mode
+
+SSO mode for C3. Possible values: oidc. if enabling oidc you must set  sso_groups, sso_sub, sso_jwks_uri, sso_authorize_uri, sso_token_uri, sso_issuer_url, sso_client_id, sso_client_password in MDS
+
+Default:  none
+
+***
+
+### sso_groups
+
+Groups in JWT
+
+Default:  none
+
+***
+
+### sso_sub
+
+Sub in JWT
+
+Default:  none
+
+***
+
+### sso_issuer_url
+
+The issuer url, which is typically the authorization server's URL
+
+Default:  none
+
+***
+
+### sso_jwks_uri
+
+JSON Web Key Set (JWKS) URI
+
+Default:  none
+
+***
+
+### sso_authorize_uri
+
+Base uri for authorize endpoint, that initiates an OAuth authorization request
+
+Default:  none
+
+***
+
+### sso_token_uri
+
+Base uri for IdP token endpoint, from where a token is requested by MDS
+
+Default:  none
+
+***
+
+### sso_client_id
+
+Client id for authorize and token request to Idp
+
+Default:  none
+
+***
+
+### sso_client_password
+
+Client password for authorize and token request to Idp
+
+Default:  none
+
+***
+
+### sso_scope
```

Comment:
Since this variable is used only to mention scope for groups. And there are many other possible scopes like "openid", "offline_access", etc., it should be renamed to include groups.

Reply:
would sso_include_groups be good?

Reply:
"scope" is well known term in OAuth. Could we use `sso_groups_scope` ?